jenkins/debian_c.sh

```diff
@@ -18,11 +18,18 @@ build_folder=$build_root"/cmake/debian_linux"
 # Set the default cores
 CORES=$(grep -c ^processor /proc/cpuinfo 2>/dev/null || sysctl -n hw.ncpu)
 
+if [ $CORES -eq 1 ]
+then
+    LOADLIMIT=$CORES
+else
+    LOADLIMIT=$(perl -e "print int($CORES * 0.8)")
+fi
+
 rm -r -f $build_folder
 mkdir -p $build_folder
 pushd $build_folder
 cmake -Drun_valgrind:BOOL=ON $build_root -Drun_unittests:BOOL=ON
-make --jobs=$CORES
+make --jobs=$CORES --max-load=$LOADLIMIT
```

Comment:
Shouldn't we put this on all our jenkins/*.sh files, just for uniformity?  Ideally with a helper .sh that had a function() to calculate that though we could take an IOU on that one given need to get this in.